doc/website/getting-started/list-of-examples.md

```diff
@@ -0,0 +1,16 @@
+# List of examples
+
+| Example                                         | Description                                                 | Level              |
+|:------------------------------------------------|:------------------------------------------------------------|:-------------------|
+|[icehello](./examples/icehello/)                 | How to send data to another process                         | :star:             |
+|[icedelivery](./examples/icedelivery/)           | How to send and receive data in a functional way            | :star:             |
+|[iceoptions](./examples/iceoptions/)             | Configure the history cache size and the starting behaviour | :star:             |
```

Comment:
It is hard to understand what is meant with "starting behaviour".
Maybe this can be more generic like:
"Extended Configuration of behavior for Publisher/Subscriber Ports"

Reply:
I'd explicitly wanted to mention an example or the point that is most important from user perspective. In terms of issues on GitHub it was definitely the "I missed the first sample in the example" issue. I suppose "Extended configuration" wouldn't guide the user to this example when reading over the list, what do you think about "Configure the history cache size and the starting behaviour of publishers and subscribers"

Reply:
I would call it "(Advanced) configuration options for ports"

Reply:
@MatthiasKillat We opted out long time ago, to call the publisher and subscribers "ports". 

I went for "Configure pub/sub settings like history cache size or startup behaviour"